src/controllers/usersController.js

```diff
@@ -19,10 +22,48 @@ exports.registeration = (req, res) => {
     res.json(err);
   });
 };
-exports.login = (req, res) => {
-// return token
-  res.json('login in progress');
-};
+
+exports.login = [
```

Comment:
the issue was for users to login with their tax payer Id and  password . not email and password

Reply:
Ok, would fix that

Reply:
Okay

Reply:
fixed